bip-0003.md

```diff
@@ -391,67 +389,69 @@ innovate on a level playing field. Only freely licensed contributions are accept
-license terms.
+It is recommended that source code included in a BIP (whether within the text or in auxiliary files) be licensed under the same license terms as the project it
+is proposed to modify, if any. For example, changes intended for Bitcoin Core would ideally be licensed (also) under the MIT
+License.
 
 In all cases, details of the licensing terms must be provided in the Copyright section of the BIP.
 
 #### Acceptable Licenses[^licenses]
 
-* BSD-2-Clause: [OSI-approved BSD 2-clause license](https://opensource.org/licenses/BSD-2-Clause)
-* BSD-3-Clause: [OSI-approved BSD 3-clause license](https://opensource.org/licenses/BSD-3-Clause)
+Each new BIP must be made available under at least one acceptable license as listed below. BIPs are not required to be
+*exclusively* licensed under approved terms, and may also be licensed under other licenses *in addition to* at least one
+acceptable license.
+
+In other words, a new BIP must specify an SPDX License Expression that is either "L" or equivalent to "L OR E" for some
+acceptable license L from the following list and another SPDX License Expression E.
+
+* BSD-2-Clause: [BSD 2-Clause License](https://opensource.org/licenses/BSD-2-Clause)
+* BSD-3-Clause: [BSD 3-Clause License](https://opensource.org/licenses/BSD-3-Clause)
 * CC0-1.0: [Creative Commons CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/)
-* GNU-All-Permissive: [GNU All-Permissive License](http://www.gnu.org/prep/maintain/html_node/License-Notices-for-Other-Files.html)
+* FSFAP: [FSF All Permissive License](https://www.gnu.org/prep/maintain/html_node/License-Notices-for-Other-Files.html)
 * CC-BY-4.0: [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/)
-* MIT: [Expat/MIT/X11 license](https://opensource.org/licenses/MIT)
-* Apache-2.0: [Apache License, version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
-* BSL-1.0: [Boost Software License, version 1.0](http://www.boost.org/LICENSE_1_0.txt)
+* MIT: [Expat/MIT/X11 License](https://opensource.org/licenses/MIT)
+* Apache-2.0: [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0)
+* BSL-1.0: [Boost Software License 1.0](https://www.boost.org/LICENSE_1_0.txt)
 
 #### Not Acceptable Licenses
 
 All licenses not explicitly included in the above lists are not acceptable terms for a Bitcoin Improvement Proposal.
-However, BIPs predating this proposal were allowed under other terms, and should use these abbreviations
-when no other license is granted:
+However, BIPs predating this proposal were accepted under other terms, and should use one the following identifiers.
 
-* PD: Released into the public domain
-* OPL: [Open Publication License, version 1.0](http://opencontent.org/openpub/)
+* LicenseRef-PD: Placed into the public domain
+* OPUBL-1.0: [Open Publication License 1.0](https://opencontent.org/openpub/)
```

Comment:
In commit 85a248f68eb9, maybe update the link https://spdx.org/licenses/OPUBL-1.0.html